random-menu-generator.rb

```diff
@@ -0,0 +1,27 @@
+# RANDOM FOOD GENERATOR
+
+puts "Please enjoy one of our following specials:"
+# Three arrays to hold food lists
+adj_one = ["healthy", "cold", "luke-warm", "pan-seared", "slow-cooked", "steamed", "peppered", "pickled", "fresh", "homemade", "yummy"]
+adj_two = ["acidic", "blazed", "burnt", "cheesy", "deep-fried", "encrusted", "fluffy", "frozen", "greesy", "infused", "marinated"]
+food_type = ["roast", "enchiladas", "pancakes", "pastry", "custard", "salad", "casserole", "pasta", "vegetables", "soup", "popsicle"]
+
+
+# Length of the arrays
+array_length = adj_one.length
+
+# Variable "i" will hold our iteration number for the loop below
+i = 1
+
+# Picks and prints food items
+while i <= array_length
+  # Shuffles old array (and saves it!), pulls the last item off and uses it as random pull
+  random_adj_one = adj_one.shuffle!.pop
```

Comment:
This is great! There is so much happening in this one line, but i'm glad you only need this one line to pull the words. One thought is that you may not necessarily need to shuffle the list every time  - since after the first time it will already be shuffled. This is more of a performance consideration because eventually we want to make sure that we are only calling the methods that we need to be calling.